src/generic_core/social.ts

```diff
@@ -758,7 +758,7 @@ export function notifyUI(networkName :string, userId :string) {
             tokenObj['userId'] = this.myInstance.userId;
             tokenObj['instanceId'] = this.myInstance.instanceId;
           }
-          return 'https://www.uproxy.org/invite/' +
+          return 'https://www.uproxy.org/invite/#' +
```

Comment:
To my mind the URLs would be neater without the trailing slash, i.e.:
```
https://www.uproxy.org/invite#bigblobofbase64
```
Is there some reason to keep the trailing slash?